json-rpc/types/src/views.rs

```diff
@@ -471,11 +471,13 @@ impl std::convert::AsRef<[u8]> for BytesView {
 
 impl std::fmt::Display for BytesView {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        for byte in self.inner() {
-            write!(f, "{:02x}", byte)?;
-        }
+        f.write_str(&hex::encode(self.inner()))
+    }
+}
```

Comment:
Would you be able to change this back? The one reason this doesn't use hex encode is in order to avoid an allocation.

Reply:
Sure I can revert it. It was just making my life hard reading through debug logs that looked like:

```
BytesView([
    123,
    42,
    // .. repeat 1000 times
    99,
    0,
])
```

Do you know if there's a way to debug display it more compactly w/o allocating?

Reply:
Oh woops I'm a dummy haha, yeah i'll keep the loop then 😅